test/test_auth_e2e.py

```
@@ -0,0 +1,490 @@
+        assert "*" not in access[0]["actions"]
+
+    def test_oauth_scope_limits_unauthorized_actions(self, app, client):
+        """
+        Test 12: OAuth token authentication works through V2 auth.
+
+        An OAuth token with repo:read scope authenticates the user and
+        the V2 auth endpoint issues a bearer token based on the user's
+        repository permissions.
+        """
+        user = get_user("devtable")
+        oauth_token_obj, oauth_token_str = create_user_access_token(
```

Ruff (0.15.20)

[warning] Unpacked variable `oauth_token_obj` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)